remoting/getty/config.go

```diff
@@ -177,6 +182,23 @@ func (c *GettySessionParam) CheckValidity() error {
 	return nil
 }
 
+func parseTimeDurationByRange(timeStr string, min *time.Duration, max *time.Duration) (time.Duration, error) {
+	result, err := time.ParseDuration(timeStr)
+	if err != nil {
+		return *min, err
+	}
+	if min != nil && max != nil && *min > *max { // swap
```

Comment:
IMO, if the value is invalid, the program should go crash to make the user realize something goes wrong, instead of swapping without any warning.